src/test/resources/samples/1.22/info/2.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
You might notice that the DriverStatus part is quite different. Took me some time to put that all in. Also the other IDs, OS and so on are different. This is a dump from my machine. the only thing changed is the host name.